pydicom/dataset.py

```diff
@@ -720,12 +721,6 @@ def elements(self):
         for tag in taglist:
             yield self.get_item(tag)
 
-    def _is_uncompressed_transfer_syntax(self):
```

Comment:
I think I'm missing something -- why is the `_is_uncompressed_transfer_syntax` removed?  Did this functionality go somewhere else?

Reply:
> why is the _is_uncompressed_transfer_syntax removed? 

Right, I forgot to mention this - it is an unused private function, so I removed it. If it is here for some future use, I can put it back, of course.